grpc/CMakeLists.txt

```diff
@@ -58,8 +58,15 @@ add_custom_command(
   COMMAND
     ${RSYNC}
       --prune-empty-dirs --archive
+      --exclude="*.cmake" --exclude="libproto*"
       ${GRPC_BUILD_INSTALL_DIR}/lib/
       ${GRPC_LIB_DESTINATION}/
+  COMMAND
+    ${RSYNC}
+      --prune-empty-dirs --archive
+      --include="*/" --include="libproto*" --exclude="*"
```

Comment:
I also found the libraries for protobuf is now located at `lib/` and they can be found with `find_package(Protobuf)` in downstream packages, which means the installation of this package can have an impact on other libraries that rely on Protobuf.
Before #51, the libraries were located at `lib/protobuf` and could not be found with `find_package(Protobuf)`, the system installed protobuf libraries were used instead, so no effect on downstream packages.

I added a commit to revert the change in #51 for that part and put all Protobuf libraries into `lib/protobuf` directory.

Reply:
I cannot run `protoc` because `protoc` cannot find `libprotoc.so`.
Could you fix it?

```
$ rosrun grpc protoc
/home/rueda/gitai/catkin_ws/devel/lib/grpc/protoc: error while loading shared libraries: libprotoc.so.3.13.0.0: cannot open shared object file: No such file or directory
```

Reply:
Thank you for checking it!
I added the `-Dprotobuf_BUILD_SHARED_LIBS=OFF` to prevent the error and checked the `protoc` works without any error.

Reply:
Do we still need `--include="libproto*"` and `--exclude=libproto*` with `-Dprotobuf_BUILD_SHARED_LIBS=OFF`?

If not, could you remove it?